src/Traits/AssertDispatchedWorkflowsOrActivities.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Workflow\Traits;
+
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\App;
+use PHPUnit\Framework\Assert as PHPUnit;
```

Comment:
There might be a problem with this because PHPUnit is usually require-dev not require and neither are actually listed in our composer.json. So we should only expect PHPUnit to be installed when running tests. I will do some more local testing to make sure.